publiq-ui/pub-pagination.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubPagination from './pub-pagination.vue';
+
+<Meta title="pub-pagination" component={PubPagination} />
+
+export const Template = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubPagination },
+    template: '<pub-pagination v-bind="$props" />',
+  };
+};
+
+# Variants
```

Comment:
As mentioned in #48, there's no reason to mention Variants here if there are none 😃 Not every components will have variants.

Reply:
Ah looks like it was fixed in a more recent commit already 👍